lib/v2/ruancan/radar.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+    'ruancan.com': {
+        _name: '软餐',
+        '.': [
+            {
+                title: '首页',
+                docs: 'https://docs.rsshub.app/new-media.html#ruan-can-shou-ye',
+                source: ['/'],
+                target: '/',
```

Comment:
```suggestion
                target: '/ruancan',
```
[script-standard](https://docs.rsshub.app/joinus/script-standard.html) has a typo in radar rules.
[quick-start](https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-radar-gui-ze) has the right format.

Reply:
thank you for your review ❤️